<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Length 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Length 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_yesno">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_checkEl">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Element</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_length">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bulean">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Bool</string>
    </property>
   </widget>
   <widget class="QPushButton" name="max">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
   <widget class="QPushButton" name="min">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bulean">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_maxNeg">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_minPos">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_addEl">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_subEl">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="substract">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>sub</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_Memo1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>210</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Set 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Set 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="union_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Union</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_length1">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="set1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Set 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Set 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="crossing">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Crossing</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="MainSet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="SecondSet">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
